### PR TITLE
Feature/entry popup

### PR DESCRIPTION
**Pull request by aemiers**
### Description
* Is this a feature or a fix?
Feature-deature!

* Where should the reviewer start?
Start with the HTML at the very bottom, this is where I added the popup forms

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Adds buttons, forms, so that user can input data. Will console.log information in the format that is needed to store the infromation in the API. Still need to grab the userID and do happy and sad paths.

* What outside features or research did you use to implement this fix?
MDN 

* How should this be tested?

### Relevant Screen Shots
***
#### Please review considerations below:
- [x] Follows Turing Style Guidelines
- [x] Changes generate no new warnings or errors
- [x] Checked code for and corrected any spelling errors
- [ ] README has been updated(if applicable)

### CODE REVIEW

**Comment by alexthompson207** [comment]
This looks great! Good job with the forms!

File: src/css/_style.scss [hunk 1]
```diff
@@ -13,6 +13,30 @@
...
+  background-color: $color-black;
+  color: $color-light-blue;
+
+  &:hover {
+    background-color: $color-light-blue;
+    color: $color-black;
+    cursor: pointer;
+  }
+}
+
```
Comment (itsnameissteven) [inline src/css/_style.scss:39]: Nice nesting

Comment (itsnameissteven) [reply src/css/_style.scss:39]: can we add a transition to the button (parent) for maybe 200-300ms??

**Review by itsnameissteven — APPROVED** [review]
Great work on this.  I think once we merge this in we should have a different scss file for the form styling and another one for components like buttons. I have some components that I've worked on that could also go into a component.scss file

**Pull request merged**